crates/indexer/src/indexer.rs

```diff
@@ -497,6 +499,17 @@ impl Web3Indexer {
 
                     total_gas_limit += tx.tx.gas_limit;
 
+                    // Update tx.log.index
+                    tx.logs = tx
+                        .logs
+                        .into_iter()
+                        .map(|mut log| {
+                            log.log_index += log_index_cursor;
```

Comment:
https://github.com/godwokenrises/godwoken-web3/pull/554/files#diff-c927eb4a334b85d9eed371193a22a8058b85e434cd8c0dda821fe7aee98c964dR493

We should merge the logic with this loop

Reply:
Updated